add blog post "OpenSearch in Hopsworks" + Authors

MagicLex opened the pull request:
### Description
Add blog post "OpenSearch in Hopsworks" + Authors
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff


By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

smacrakis reviewed (commented):
Line 50 should be removed ("div>").
The section "Enabling RAG for LLMs with OpenSearch" should mention how OpenSearch is a feature-complete search solution, offering not just vector search but also other, complementary technologies.

MagicLex commented:
Thank you for your feedback.

- Removed the stray "div>" tag.
- Added information about OpenSearch's potential for hybrid search in RAG applications.
- Kept the description general to avoid overstating our current implementation in Hopsworks.

pajuric commented:
Closing due to inactivity.
